packages/apostrophe/modules/@apostrophecms/layout-widget/views/column.html

```diff
@@ -1,19 +1,27 @@
 {# Included directly from the `area.html` template.
 The area related `data.*` is available plus `item` and `widgetOptions`. #}
 {%- set lastId = apos.modules['@apostrophecms/layout-widget'].detectLastTabletFullWidthItem(data.area.items) -%}
+
+{% set styles = apos.styles.render(item) %}
+{% set styleTag = apos.styles.elements(styles) %}
+{% set wrapperAttrs = apos.styles.attributes(styles) %}
+{{ apos.log('styles', styles) }}
+{{ apos.log('wrapperAttrs', wrapperAttrs) }}
 <div
+  {{ wrapperAttrs | safe }}
   {% if data.canEdit %} data-apos-widget="{{ item._id }}"{% endif %}
   data-visible-tablet="{{ item.tablet.show }}"
   data-visible-mobile="{{ item.mobile.show }}"
   {% if lastId and item._id === lastId %}data-tablet-full="true"{% endif %}
   style="
-    --colstart: {{ item.desktop.colstart }}; 
-    --colspan: {{ item.desktop.colspan }}; 
-    --rowstart: {{ item.desktop.rowstart }}; 
-    --rowspan: {{ item.desktop.rowspan }}; 
+    --colstart: {{ item.desktop.colstart }};
+    --colspan: {{ item.desktop.colspan }};
+    --rowstart: {{ item.desktop.rowstart }};
+    --rowspan: {{ item.desktop.rowspan }};
     --justify: {{ item.desktop.justify }};
     --align: {{ item.desktop.align }};
     --order: {{ item.desktop.order }};
 ">
+  {{ styleTag | safe }}
```

Comment:
In the original auto-wrapper implementation, we put the style tag always before the wrapper. Let's keep it consistent. 
Also no need to use ` | safe` on any styles helpers, it's already done at a helper level.